### PR TITLE
Increase coverage.

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

**Comment by zxqfox** [comment]
А может не будем использовать геттеры?

File: test/type.test.js [hunk 1]
```diff
@@ -2,14 +2,19 @@ import test from 'ava';
 import sinon from 'sinon';
 import proxyquire from 'proxyquire';
 
-const spy = sinon.spy();
-const BemEntity = proxyquire('../index', {
-    'bem-naming': {
-        typeOf: spy
-    }
+test.beforeEach(t => {
+    t.context.spy = sinon.spy();
```
Comment (blond) [inline test/type.test.js]: Why not `t.spy`?

File: test/type.test.js [hunk 2]
```diff
@@ -2,14 +2,19 @@ import test from 'ava';
...
+    t.context.BemEntity = proxyquire('../index', {
```
Comment (blond) [inline test/type.test.js]: Why not `t. BemEntity`?

File: test/type.test.js [hunk 3]
```diff
@@ -2,14 +2,19 @@ import test from 'ava';
 import sinon from 'sinon';
 import proxyquire from 'proxyquire';
 
-const spy = sinon.spy();
-const BemEntity = proxyquire('../index', {
-    'bem-naming': {
-        typeOf: spy
-    }
+test.beforeEach(t => {
```
Comment (blond) [inline test/type.test.js]: May be add `setup` title?

```js
test.beforeEach('setup', t => {
```

File: test/type.test.js [hunk 4]
```diff
@@ -2,14 +2,19 @@ import test from 'ava';
...
+            typeOf: t.context.spy
+        }
+    });
 });
 
+
 test('should use `naming.typeOf()` for block', t => {
+    const BemEntity = t.context.BemEntity;
+    const spy = t.context.spy;
     const entity = new BemEntity({ block: 'block' });
```
Comment (blond) [inline test/type.test.js]: I don't like the same code in different tests. May be use `t.BemEntity` and `t.spy`?

```js
const entity = new t.BemEntity({ block: 'block' });

t.ok(t.spy.calledWith({}))
```

File: test/type.test.js [hunk 5]
```diff
@@ -44,3 +55,22 @@ test('should use naming.typeOf() for element modifier', t => {
...
+
+test('should cache type value', t => {
+    const stub = sinon.stub();
+    const BemEntity = proxyquire('../index', {
+        'bem-naming': {
+            typeOf: stub
+        }
+    });
+
+    stub.returns('type');
```
Comment (blond) [inline test/type.test.js]: Why did't you use the spy?

**Comment by blond** [comment]
I wrote some comments. @Yeti-or could you fix its?

**Comment by blond** [comment]
Closed in favor of #8

**Pull request closed without merging**